### PR TITLE
Adding Resnet

### PR DESCRIPTION
**Pull request by ymt123**
Moving the refactored classes to a different folder and adding ResNet50 features.

### CODE REVIEW

File: pelops/features/resnet50.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        if not resnet_model:
+            # include_top needs to be True for this to work
+            base_model = ResNet50(weights='imagenet', include_top=True)
+            resnet_model = Model(input=base_model.input,
+                      output=base_model.get_layer('flatten_1').output)
+
+        self.resnet_model = resnet_model
+
+    @staticmethod
+    def preprocess_image(img, x_dim=224, y_dim=224):
```
Comment (d-grossman) [inline pelops/features/resnet50.py:29]: should probably be in an external helper file, with x_dim, y_dim stored inside the class

File: pelops/features/resnet50.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        x = preprocess_input(x)
+        return x
+
+    def produce_features(self, chip):
+        pil_image = self.get_image(chip)
+        preprocessed_image = self.preprocess_image(pil_image)
+        image_features = self.resnet_model.predict(preprocessed_image)
+        return image_features
+
+    def set_variables(self):
```
Comment (d-grossman) [inline pelops/features/resnet50.py:42]: self.x_dim = 224
self.y_dim = 224

File: pelops/features/resnet50.py (hunk 1 above)
Comment (ymt123) [reply pelops/features/resnet50.py:29]: I wasn't sure because the first part could be but the 2nd (preprocess_image) is unique to ResNet.

**Comment by agude** [comment]
I'm going to merge it into my main HOG branch and we can discuss any further points on there: https://github.com/Lab41/pelops/pull/50

**Pull request merged**